cypress/e2e/04-conduct_test.cy.js

```diff
@@ -30,31 +41,52 @@ describe("Conducting a COVID test", () => {
         .contains("Begin test")
         .click();
     });
+
     cy.get(".ReactModal__Content").contains(
       "Are you experiencing any of the following symptoms?"
     );
 
     // Test a11y on the AoE modal
     cy.checkA11y();
   });
-  it("fills out the pretest questions and submits", () => {
+  it("fills out the aoe questions and submits", () => {
     cy.get(".ReactModal__Content").within(() => {
       cy.get('input[name="no_symptoms"][value="no"]+label').click();
       cy.get('input[name="pregnancy"][value="60001007"]+label').click();
       cy.get("#aoe-form-save-button").click();
     });
+
+    cy.wait("@AddPatientToQueue");
+    cy.wait("@GetFacilityQueue", {timeout: 20000});
```

Comment:
`GetFacilityQueue` is on a 10 second update interval, thus it needed an extended timeout window to avoid edge cases